x-pack/plugins/enterprise_search/public/plugin.ts

```diff
@@ -470,33 +469,29 @@ export class EnterpriseSearchPlugin implements Plugin {
       title: ANALYTICS_PLUGIN.NAME,
     });
 
-    this.licenseSubscription = licensing?.license$.subscribe((license: ILicense) => {
-      if (license.isActive && license.hasAtLeast('enterprise')) {
-        core.application.register({
-          appRoute: INFERENCE_ENDPOINTS_PLUGIN.URL,
-          category: DEFAULT_APP_CATEGORIES.enterpriseSearch,
-          deepLinks: relevanceLinks,
-          euiIconType: INFERENCE_ENDPOINTS_PLUGIN.LOGO,
-          id: INFERENCE_ENDPOINTS_PLUGIN.ID,
-          mount: async (params: AppMountParameters) => {
-            const kibanaDeps = await this.getKibanaDeps(core, params, cloud);
-            const { chrome, http } = kibanaDeps.core;
-            chrome.docTitle.change(INFERENCE_ENDPOINTS_PLUGIN.NAME);
-
-            await this.getInitialData(http);
-            const pluginData = this.getPluginData();
-
-            const { renderApp } = await import('./applications');
-            const { EnterpriseSearchRelevance } = await import(
-              './applications/enterprise_search_relevance'
-            );
-
-            return renderApp(EnterpriseSearchRelevance, kibanaDeps, pluginData);
-          },
-          title: INFERENCE_ENDPOINTS_PLUGIN.NAV_TITLE,
-          visibleIn: [],
-        });
-      }
```

Comment:
Since we do not use **this.licenseSubscription** any more, we may need to clean up the related codes:

- [line1](https://github.com/elastic/kibana/blob/fc46cc30885c5c60a05d64ca7cdb22a2a1c878dc/x-pack/plugins/enterprise_search/public/plugin.ts#L191)
- [line2](https://github.com/elastic/kibana/blob/fc46cc30885c5c60a05d64ca7cdb22a2a1c878dc/x-pack/plugins/enterprise_search/public/plugin.ts#L677-L679)